### PR TITLE
Attempts to resolve aarch64 CI issues

### PR DESCRIPTION
**Pull request by BradyBonnette**

### CODE REVIEW

**Review by thomcc — APPROVED** [review]

**Comment by thomcc** [comment]
I'm fine with this but it should not make a difference.

**Comment by thomcc** [comment]
I think we need to do this in a single `run:` It seems you're right that the env setting doesn't work for self-hosted runners.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -76,19 +78,25 @@ jobs:
...
         echo "RUSTUP_HOME=$HOME/.rustup" >> $GITHUB_ENV
         echo "PATH=$HOME/.cargo/bin:$PATH" >> $GITHUB_ENV
+        cargo --version
 
     - name: Update postgrestd git submodules
       run: git submodule update --init --recursive
 
     - name: Build postgrestd
-      run: ./run install
+      # run: STD_TARGETS=aarch64-postgres-linux-gnu ./run install
```
Comment (thomcc) [inline .github/workflows/main.yml]: Let's get a comment on why we are doing it like this.

Comment (BradyBonnette) [reply .github/workflows/main.yml]: Ok, Ill add some comments when I go to clean it up

**Review by thomcc — APPROVED** [review]

**Pull request merged**